harbor-ui/src/components/sidebar.rs

```diff
@@ -12,7 +12,8 @@ use super::{harbor_logo, indicator, lighten, map_icon, sidebar_button};
 
 pub fn sidebar(harbor: &HarborWallet) -> Element<Message> {
     // Transfers require at least 2 federations
-    let transfer_disabled = harbor.federation_list.len() < 2;
+    let transfer_disabled =
```

Comment:
i like keeping it at just 2, makes it simpler

Reply:
if you have 2 federations but no balances then you can go to the transfer screen but you can't do anything on the transfer screen because you're poor

Reply:
yeah i feel ike people just want to poke around and see it though